pkgs/applications/audio/sonixd/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
+
+let
+  pname = "sonixd";
+  version = "0.14.0";
+  name = "sonixd-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/jeffvli/sonixd/releases/download/v${version}/Sonixd-${version}-linux-x86_64.AppImage";
+    sha256 = "sha256-q+26Ut5wN9gFDBdqirR+he/ppu/b1wiqq23WkcRAQd4=";
+  };
+
+in
+appimageTools.wrapType2 {
+  inherit name src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
```

```suggestion
    mv $out/bin/sonixd-${version} $out/bin/sonixd
```
personally I prefer it this way, because the only thing that will change is the version